remo/events/api/views.py

```diff
@@ -19,11 +19,23 @@
 KPI_WEEKS = 12
 
 
+class EventsFilter(django_filters.FilterSet):
+    owner = django_filters.CharFilter(name='owner__userprofile')
+    categories = django_filters.CharFilter(name='categories__name')
+    initiative = django_filters.CharFilter(name='campaign__name')
+
+    class Meta:
+        model = Event
+        fields = ('name', 'description', 'start', 'end', 'city', 'region',
```

Comment:
Just reference the bug here, when it becomes available so that we can track why these filters aren't implemented here.